internal/metrics/metrics.go

```diff
@@ -171,6 +173,13 @@ func NewMetrics(registry *prometheus.Registry) *Metrics {
 			Help:       "Histogram for the runtime of xDS cache regeneration.",
 			Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001},
 		}),
+		EventHandlerOperations: prometheus.NewCounterVec(
+			prometheus.CounterOpts{
+				Name: eventHandlerOperations,
+				Help: "Total number of ResourceEventHandler operations by operation and object kind",
```

Comment:
Not sure that operators will know what to make of "ResourceEventHandler operations". Maybe

```
Total number of Kubernetes object changes Contour has received by operation and object kind.
```

Reply:
That's really good.

Reply:
done